Purchases/Public/RCPurchases.m

```diff
@@ -539,23 +550,26 @@ - (void)updateEntitlementsCache:(RCReceiveEntitlementsBlock _Nullable)completion
 
 - (void)receiptData:(void (^ _Nonnull)(NSData * _Nonnull data))completion
 {
-    NSURL *receiptURL = [[NSBundle mainBundle] appStoreReceiptURL];
-    NSData *receiptData = [NSData dataWithContentsOfURL:receiptURL];
-    RCDebugLog(@"Loaded receipt from %@", receiptURL);
+    NSData *receiptData = [self.receiptFetcher receiptData];
     if (receiptData == nil) {
         RCDebugLog(@"Receipt empty, fetching");
-        [self.requestFetcher fetchReceiptData:^{
-            NSData *newReceiptData = [NSData dataWithContentsOfURL:receiptURL];
-            if (newReceiptData == nil) {
-                RCLog(@"Unable to load receipt, ensure you are logged in to a Sandbox account");
-            }
-            completion(newReceiptData ?: [NSData data]);
-        }];
+        [self refreshReceipt:completion];
     } else {
         completion(receiptData);
     }
 }
 
+- (void)refreshReceipt:(void (^ _Nonnull)(NSData * _Nonnull data))completion
+{
+    [self.requestFetcher fetchReceiptData:^{
+        NSData *newReceiptData = [self.receiptFetcher receiptData];
+        if (newReceiptData == nil) {
+            RCLog(@"Unable to load receipt, ensure you are logged in to a Sandbox account");
```

Comment:
This log message could happen on production and would be misleading.

> Unable to load receipt, ensure you are logged in to the correct iTunes account.

Might be a better log.